packages/carousel/src/react/context.ts

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+interface CarouselContextValue {
+  activePage: number
+  next: () => void
+  offset: number
+  prev: () => void
+  id: string
+  pageCount: number
+  perPage: number
+  transitioning: boolean
+  setTransitioning: React.Dispatch<React.SetStateAction<boolean>>
+}
+const context = React.createContext<CarouselContextValue>({
+  activePage: 0,
```

Comment:
Some of these defaults are new.  Simplified the typing process. Double-check my logic that having these are ok as defaults.